pbjson-build/src/generator/message.rs

```diff
@@ -576,6 +611,7 @@ fn write_deserialize_field_name<W: Write>(
 {indent}                write!(formatter, "expected one of: {{:?}}", &FIELDS)
 {indent}            }}
 
+{indent}            #[allow(unused_variables)]
```

Comment:
I'm guessing this is needed to handle the case of no fields?

Reply:
Yes, the build was failing because of that, we could add an if, to use underscore when needed, but was hesitant to add more branching that may make the code a little bit more confusing, so this sounded like a good compromise.

Reply:
Makes sense :+1: